app/assets/javascripts/application.js

```diff
@@ -9,6 +9,7 @@
 //= require external_link
 //= require image_vote
 //= require image_export
+//= require visual_group_status
```

Comment:
Rather than requiring `visual_group_status` on all pages, consider using `javascript_include` to include it only in the templates where it's needed. 

Check the observations/show template for an example (`suggestions` and `vote_popup`) or pivotal/index.

Reply:
Done.